spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1385,23 +1385,25 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     testDoubleScalarExpr("expm1")
   }
 
-  // https://github.com/apache/datafusion-comet/issues/666
-  ignore("abs") {
-    Seq(true, false).foreach { dictionaryEnabled =>
-      withTempDir { dir =>
-        val path = new Path(dir.toURI.toString, "test.parquet")
-        makeParquetFileAllPrimitiveTypes(path, dictionaryEnabled = dictionaryEnabled, 100)
-        withParquetTable(path.toString, "tbl") {
-          Seq(2, 3, 4, 5, 6, 7, 9, 10, 11, 12, 15, 16, 17).foreach { col =>
-            checkSparkAnswerAndOperator(s"SELECT abs(_${col}) FROM tbl")
+  test("abs") {
+    Seq(true, false).foreach { ansi_enabled =>
+      Seq(true, false).foreach { dictionaryEnabled =>
+        withSQLConf(SQLConf.ANSI_ENABLED.key -> ansi_enabled.toString) {
```

Comment:
The test isn't currently testing any scenarios where ANSI mode would cause an exception to be thrown.